lnrpc/devrpc/log.go

```diff
@@ -0,0 +1,48 @@
+//go:build dev
+// +build dev
```

Comment:
I think for the logging to work we'll need to remove the build tags and also add the `devrpc.UseLogger` to the root `log.go`.

Reply:
Done.